CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased]
+## [3.0.0a0] - 2024-05-10
```

Comment:
There are links at the bottom of this page that need to be updated.

Reply:
https://github.com/stac-utils/stac-fastapi-pgstac/blob/9163c6bb240a62ab9975bab420fd4ffb0e757d9c/CHANGES.md?plain=1#L309-L311